src/tests/ftest/dfuse/bash.py

```diff
@@ -17,14 +17,9 @@ class Cmd(DfuseTestBase):
     :avocado: recursive
     """
 
-    def test_bashcmd(self):
+    def run_bashcmd(self, il_lib=None, compatible_mode=False):
```

Comment:
> GitHub Actions / Pylint check
> too-many-locals, Too many local variables (34/30)

Reply:
Please fix this.

Reply:
ok. I will fix it. Thanks, @daltonbohning and @ashleypittman!